src/pages/Vaults/Vault/index.tsx

```diff
@@ -262,7 +290,18 @@ const Vault = (): JSX.Element => {
       <MainContainer className='fade-in-animation'>
         <VaultsHeader title={t('vault.vault_dashboard')} accountAddress={selectedVaultAccountAddress} />
         <div className='space-y-6'>
-          <SectionTitle>Vault Stats</SectionTitle>
+          <SectionTitle>
+            Vault Stats: {collateralCurrencyValues?.id}/{WRAPPED_TOKEN_SYMBOL}{' '}
+          </SectionTitle>
+          <Panel className={clsx('inline-block', 'px-4', 'py-2')}>
+            <Dl
+              listItems={[
+                { term: 'Liquidation threshold', definition: `${liquidationThreshold}%` },
+                { term: 'Premium redeem threshold', definition: `${premiumRedeemThreshold}%` },
+                { term: 'Ideal threshold', definition: `${secureThreshold}%` }
+              ]}
```

Comment:
I just noticed the `Dl` component from our library is not very responsive, I think we should update it:

![Screen Shot 2022-08-15 at 15 48 52](https://user-images.githubusercontent.com/47864599/184647703-1952837d-23f6-4f26-9308-e4197b3b1557.png)

Reply:
Yes :) will check in with Jay, but this will be fixed for the new vault dashboard